opentelemetry/proto/profiles/v1experimental/pprofextended.proto

```diff
@@ -123,6 +123,14 @@ message Profile {
   // Index into the string table of the type of the preferred sample
   // value. If unset, clients should default to the last sample value.
   int64 default_sample_type = 14;
+  // Documentation link for this profile. The URL must be absolute,
+  // e.g., http://pprof.example.com/cpu-profile.html
+  //
+  // The URL may be missing if the profile was generated by older code or code
+  // that did not bother to supply a link.
+  int64 doc_url = 19;  // Index into string table.
```

Comment:
With `ScopeProfiles.schema_url` the URL that describes the provided profile exists already. Where do you see the differences between the two, so that this additional field is required?

Reply:
schema_url points to https://opentelemetry.io/docs/specs/otel/schemas/#schema-url file which is not user docs. The doc_url here is for user-level documentation. I would also expect that the schema URL would be the same for many profiles as it corresponds to the version of the data schema or something like that rather than the nature of the data (CPU profile vs heap profile vs contention profile).

Reply:
Since this should be an optional field, what do you think about using an attribute? 

On a similar note, we had a discussion in the last SIG about the boolean [fields](https://github.com/open-telemetry/opentelemetry-proto/blob/6f589125b0b7d708c9b0f32916378182ac1e123d/opentelemetry/proto/profiles/v1experimental/pprofextended.proto#L319-L322) in `Mapping` and the consensus was to drop from the proto and turn them into attributes.

Reply:
I think the doc_url can be an attribute. I'll change this.

For the has_* fields I'm less sure. I added an agenda topic to the SIG notes doc with some thoughts on this.